Instagram.js

```diff
@@ -135,7 +135,7 @@ export default class Instagram extends Component {
   }
 
   renderWebview() {
-    const { appId, appSecret, redirectUrl, scopes, responseType,language='en' } = this.props;
+    const { appId, appSecret, redirectUrl, scopes, responseType,language='en', incognito='false' } = this.props;
```

Comment:
@Dlas1212  I think it's boolean, not string. please remove ''

Reply:
updated